tests/conftest.py

```diff
@@ -21,6 +21,25 @@
 from ploomber import Env
 import pandas as pd
 from glob import iglob
+from ploomber.cli import install
+from unittest.mock import Mock, MagicMock
+from ploomber.telemetry import telemetry
+
+
+@pytest.fixture(scope='class')
+def monkeypatch_session():
+    from _pytest.monkeypatch import MonkeyPatch
+    m = MonkeyPatch()
+    yield m
+    m.undo()
+
+
+@pytest.fixture(scope='class', autouse=True)
```

Comment:
Im not sure if the scope='class' is required, maybe it's enough with the autouse?

Reply:
Same as above.